en/sync-themes-plugin.md

```diff
@@ -18,17 +18,20 @@ Once a design in Sketch is created with the Indigo.Design System, use the plugin
 
 ## 1. Installing the Indigo.Design plugin
 
-The Indigo.Design Plugin is available for download from [Indigo.Design Cloud](https://cloud.indigo.design) after signing in from the `Design System` tab of the dialog that shows upon clicking the `Getting Started` button in the header. Once you download the archive and open it, double click on the Sketch plugin file and the Indigo.Design plugin will be installed.
+The Indigo.Design Plugin is available after signing in to the [Indigo.Design Cloud](https://cloud.indigo.design). Go to the `Getting Started` menu and click the `GET INDIGO.DESIGN SYSTEM` button to open the assets dialog. Under the Sketch tab you may download the plugin archive, open it, double click on the Sketch plugin file and the Indigo.Design plugin will be installed. Under the Adobe XD tab you will be able to land on the Adobe Exchange page of Indigo.Design from where you can easily get the plugin. 
 
 <img class="responsive-img" src="images/indigo-design-resources.gif" />
-<p style="text-align:center;">Indigo.Design resources</p>
 
 ## 2. Create an application in AppBuilder
-Take your designs and create interactive, responsive apps with real UI components and styling. Use the `Create App` tab and the plugin will transform your Sketch design into a real application under the AppBuilder, allowing access to the images and themes that you have used in Sketch - all of this directly in the cloud.
+Take your designs and create interactive, responsive apps with real UI components and styling. Use the `Create New App` feature of the plugin to transform your Sketch and Adobe XD designs into a real applications under the AppBuilder, allowing access to the images and themes - all of this directly in the cloud.
 
 The plugin allows you to choose a name and resolution for your app, as well as to select the Indigo.Design Cloud workspace, where the app will be created.
 
-<img class="responsive-img" src="images/create-from-sketch.gif" srcset="images/create-from-sketch.gif 2x" />
+<img class="responsive-img" src="images/create-from-sketch.gif" />
```

Comment:
this is what I got:
https://user-images.githubusercontent.com/7037740/153202155-298fdc44-e806-4a12-b60f-dd52c03e038e.mov

Reply:
Yes that is the gif at the end of section 1 and before section 2 starts, but you have tagged the comment inside section 2 that shows the correct gif
<img width="1090" alt="Screenshot 2022-02-09 at 15 05 34" src="https://user-images.githubusercontent.com/1702064/153207058-34843c8a-aef2-4faf-8255-a71b590300d9.png">
s

Reply:
Yeah, my bad. I meant the first gif showing the installation process. It is saying that it will show how to download the sketch plugin but it shows how to download the xd plugin.